addOns/retest/src/main/java/org/zaproxy/addon/retest/EditAlertDialog.java

```diff
@@ -0,0 +1,170 @@
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2021 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.retest;
+
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.HeadlessException;
+import javax.swing.JButton;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.extension.AbstractDialog;
+
+public class EditAlertDialog extends AbstractDialog {
+
+    private static final Logger logger = LogManager.getLogger(EditAlertDialog.class);
+    private static final long serialVersionUID = 1L;
+
+    private JPanel jPanel = null;
+    private EditAlertPanel alertEditPanel = null;
+    private JButton btnOk = null;
+    private JButton btnCancel = null;
+    private AlertData alertData;
+    private PlanTableModel tableModel = null;
+    private int rowIdx;
+
+    public EditAlertDialog(PlanTableModel tableModel, int rowIdx) throws HeadlessException {
+        this.tableModel = tableModel;
+        this.rowIdx = rowIdx;
+        this.alertData = tableModel.getRow(rowIdx);
+
+        this.setTitle(Constant.messages.getString("retest.edit.dialog.title"));
+        this.setContentPane(getJPanel());
+        this.getAlertEditPanel().displayAlert(alertData);
+        this.addWindowListener(
+                new java.awt.event.WindowAdapter() {
+                    @Override
+                    public void windowOpened(java.awt.event.WindowEvent e) {}
+
+                    @Override
+                    public void windowClosing(java.awt.event.WindowEvent e) {
+                        btnCancel.doClick();
+                    }
+                });
+        pack();
+    }
+
+    /**
+     * This method initializes jPanel
+     *
+     * @return javax.swing.JPanel
+     */
+    private JPanel getJPanel() {
+        if (jPanel == null) {
+            GridBagConstraints gridBagConstraints15 = new GridBagConstraints();
+            java.awt.GridBagConstraints gridBagConstraints13 = new GridBagConstraints();
+
+            javax.swing.JLabel jLabel2 = new JLabel();
+
+            java.awt.GridBagConstraints gridBagConstraints3 = new GridBagConstraints();
+
+            java.awt.GridBagConstraints gridBagConstraints2 = new GridBagConstraints();
```

Comment:
These declarations shouldn't need to be fully qualified, also the blank lines can be removed.